pyproject.toml

```diff
@@ -28,7 +28,7 @@ dependencies = [
     "frequenz-api-microgrid >= 0.11.0, < 0.12.0",
     "frequenz-channels >= 0.11.0, < 0.12.0",
     "google-api-python-client >= 2.71, < 3",
-    "grpcio >= 1.51.1, < 2",
+    "grpcio == 1.51.1",
```

Comment:
Can you maybe split all the pinning (here and in `noxfile.py`) to a separate commit to make it more clear (I guess this is to make the CI pass?).

We probably should consider start pinning the dev dependencies always, so we don't have to do this when we need to release a hot fix for an old version.

Reply:
True :) Thanks. I separated commits.